sql/updates/world/3.3.5/9999_99_99_99_world_WARDEN.sql

```diff
@@ -0,0 +1,17 @@
+-- 
+ALTER TABLE `warden_checks`
+    MODIFY COLUMN `str` VARCHAR(170) DEFAULT NULL,
+    MODIFY COLUMN `result` VARBINARY(24) DEFAULT NULL;
+
+-- shift all warden checks down by 1 so we no longer have a gap at id 0
+-- (left join is for reapply safety)
+UPDATE `warden_checks` wc1
+    LEFT JOIN `warden_checks` wc2 ON `wc2`.`id` = 0
+    SET `wc1`.`id` = `wc1`.`id`-1
+    WHERE `wc2`.`id` IS NULL;
+
+DELETE FROM `warden_checks` WHERE `id` IN (786,787,788);
+INSERT INTO `warden_checks` (`id`,`type`,`str`,`address`,`length`,`result`, `comment`) VALUES
+(786, 139, 'forceinsecure() return issecure()', NULL, NULL, NULL, 'Detects naive Lua unlockers'),
+(787, 243, NULL, CONV('819210',16,10), 16, UNHEX('558BEC518305A013D40001A19C13D400'), 'FrameScript::Execute'),
```

Comment:
you know you can just write 0x819210 and mysql will insert it properly?

Reply:
for binary i personally do `x'CAFEBABE'` and it just works